Python/omevv/v1/scripts/GroupFirmwareInventory.py

```diff
@@ -0,0 +1,82 @@
+from typing import Any, Dict, List, Optional, Union
+import argparse
+from omevv_apis_client import AuthenticatedClient
+from omevv_apis_client.api.group_inventory import get_managed_hosts_firmware_inventory_by_group
+from omevv_apis_client.models import ManagedHost
+from omevv_apis_client.models import GroupFirmwareInventoryCollectionModel
+from omevv_apis_client.models import ErrorObject
+from omevv_apis_client.models import Credential
+import constants
+import base64
+import pandas as pd
```

Comment:
why do we need this?

Reply:
@pavithra-mahadev : Removed the imports not using it in the scripts.